settings.gradle

```diff
@@ -41,3 +37,8 @@ boolean omnitureDependenciesAreSatisfied() {
     File dep = file("${System.getProperty("user.home")}/libs/adobeMobileLibrary.jar")
     return dep.exists()
 }
+
+
+// For building exoplayer-specific sample apps, which are located under the 'exoplayer' directory.
+include ':ExoPlayerSampleApp'
+project(':ExoPlayerSampleApp').projectDir = new File(settingsDir, 'exoplayer/ExoPlayerSampleApp')
```

Comment:
:exoplayer:ExoPlayerSampleApp didn't work. It kept thinking that the exoplayer subdirectory was an android project and kept looking for a manifest.